src/app/business/home/home.component.ts

```diff
@@ -83,16 +84,26 @@ export class HomeComponent implements OnInit {
         
         this.getCounts();
         this.getType();
-        
-        this.backendForm = this.fb.group({
+        if(this.isLocalCloud){this.backendForm = this.fb.group({
```

Comment:
@Click2Cloud-Gamma since only one field is different in this entire block, the if condition can change only the `region` field.